src/ContaoManager/Plugin.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Terminal42\ContaoAutoRegistrationBundle\ContaoManager;
+
+use Contao\CoreBundle\ContaoCoreBundle;
+use Contao\ManagerPlugin\Bundle\BundlePluginInterface;
+use Contao\ManagerPlugin\Bundle\Config\BundleConfig;
+use Contao\ManagerPlugin\Bundle\Config\ConfigInterface;
+use Contao\ManagerPlugin\Bundle\Parser\ParserInterface;
+use Terminal42\ContaoAutoRegistrationBundle\Terminal42ContaoAutoRegistrationBundle;
+
+/**
+ * Contao Manager plugin.
```

Comment:
No need for any class docblocks if they don't provide useful information (like `@param`).